windows_containers/index.adoc

```diff
@@ -8,6 +8,13 @@ toc::[]
 
 {productwinc} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. For more information, see the xref:../windows_containers/windows-containers-release-notes-5-x.adoc#windows-containers-release-notes-5-x[release notes].
 
+Windows nodes can be added either by creating a xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[machine set] or by specifying existing Bring-Your-Own-Host (BYOH) Window instances through a xref:../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[configuration map].
+
+[NOTE]
+====
+Machne sets are not supported for bare metal/provider agnostic clusters.
```

Comment:
```suggestion
Machine sets are not supported for bare metal or provider agnostic clusters.
```

Reply:
@kalexand-rh Sorry! I must have missed pushing that change. Thanks!

Reply:
No worries! I'm glad you were on it!